science/ALPSCore/Portfile

```diff
@@ -19,8 +19,8 @@ long_description    ALPSCore contains the core libraries of the applications and
 
 homepage            http://alpscore.org
 
-checksums           rmd160  bf0fac92dfaa831379198d19e1593a241378841a \
-                    sha256  5468f73a11e603cdfe7e99ec0b5cbb55915f7fadfc04201e6c97beaa16aed4b7
+checksums           rmd160  7ef9f655f55a21658da9d7af00025d2eb7ab5a9a \
+                    sha256  01ae0c7be844d4497da96bd8b329bced3eeb4dce1101f193206e9b1c0c68068f
```

Comment:
Please add size to the checksums. This can be generated using the 'port checksums' command.